src/utils.rs

```diff
@@ -198,12 +198,11 @@ pub mod test_utils {
     macro_rules! ids {
         ( $( $name: expr ),* ) => {
             {
-                let mut ids = HashMap::<String, BigInt>::new();
+                let mut ids = HashMap::<String, usize>::new();
                 let mut num = -1;
                 $(
                     num += 1;
-                    ids_inner!($name, num, ids);
-
+                    ids_inner!($name, num.try_into().unwrap(), ids);
```

Comment:
Is there any possibility to avoid the unwrap? Like we ended up doing with the bignint macro:
`macro_rules! bigint {
    ($val : expr) => {
        Into::<BigInt>::into($val)
    };
}`

Reply:
I have tested that and I couldn't find a quick way to avoid the unwrap. This macro is only used in tests, so I wouldn't worry too much anyway, but I could open an issue to handle that later on

Reply:
Okay, maybe an issue would be good for the time being. Perhaps with all the refactoring, we won't even need to worry about this